crypto/bn/asm/x86_64-mont5.pl

```diff
@@ -3742,6 +3759,7 @@
 	lea	(%r10),%rsp
 	ret
 .LSEH_end_bn_gather5:
+.cfi_endproc
```

Comment:
The .LSEH_end_bn_gather5 seems to be unused. I find it rather strange after the ret.

Reply:
I did not check, just assumed that is used for windows seh?

Reply:
it seems to be used in windows at line 3915
```
$code.=<<___;
        .rva    .LSEH_begin_bn_gather5
        .rva    .LSEH_end_bn_gather5
        .rva    .LSEH_info_bn_gather5
```

Reply:
no idea if the windows seh handling is sane or not...